src/ontology/cl-edit.owl

```diff
@@ -4895,7 +4895,7 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000157 obo:CL_0000151
 
 # Class: obo:CL_0000158 (club cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1183/09031936.00146609") Annotation(oboInOwl:hasDbXref "DOI:https://doi.org/10.1378/chest.12-2762") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:28128362") Annotation(oboInOwl:hasDbXref "PMID:29874100") Annotation(oboInOwl:hasDbXref "PMID:7905712") obo:IAO_0000115 obo:CL_0000158 "A non-mucous, epithelial secretory cell that is part of the tracheobronchial tree. A club cell has short microvilli but no cilia. A club cell is able to multiply and differentiate into ciliated cells to regenerate the bronchiolar epithelium and it also protects the tracheobronchial epithelium.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1183/09031936.00146609") Annotation(oboInOwl:hasDbXref "DOI:10.1378/chest.12-2762") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:28128362") Annotation(oboInOwl:hasDbXref "PMID:29874100") Annotation(oboInOwl:hasDbXref "PMID:7905712") obo:IAO_0000115 obo:CL_0000158 "A non-mucous, epithelial secretory cell that is part of the tracheobronchial tree. A club cell has short microvilli but no cilia. A club cell is able to multiply and differentiate into ciliated cells to regenerate the bronchiolar epithelium and it also protects the tracheobronchial epithelium.")
```

Comment:
Should `DOI:` be `doi:` ?

Reply:
I've seen both in CL.
[Line 84 in this PR](https://github.com/obophenotype/cell-ontology/pull/2010/files) for editor guidance shows "doi:". 

@dosumis, is there a preference?

Can we merge this PR and address in a separate issue dedicated to annotation standards?

Reply:
Cross-references to DOIs in FBbt use `doi:` (lowercase) exclusively, not `DOI:`. This is [the recommendation of the DOI specification](https://www.doi.org/the-identifier/resources/handbook/2_numbering#2.6.1). No changes required.